content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -675,3 +644,33 @@ supports your chosen platform.
 If you are running into difficulties with kubeadm, please consult our
 [troubleshooting docs](/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/).
 
+
+<!-- discussion -->
+
+## What's next {#whats-next}
+
+* Verify that your cluster is running properly with [Sonobuoy](https://github.com/heptio/sonobuoy)
```

Comment:
Since the purpose of this PR is to move the What's next section, this is a non-blocking nit.
I believe we should have the third party 
```suggestion
{{% thirdparty-content single="true" %}}
* Verify that your cluster is running properly with [Sonobuoy](https://github.com/heptio/sonobuoy)
```